src/java.naming/share/classes/module-info.java

```diff
@@ -36,21 +36,31 @@
  * The following implementation specific environment properties are supported by the
  * default LDAP Naming Service Provider implementation in the JDK:
  * <ul>
+ *     <li>{@code java.naming.ldap.factory.socket}:
+ *         <br>The value of this environment property specifies the fully
+ *         qualified class name of the socket factory used by the LDAP provider.
+ *         This class must implement the javax.net.SocketFactory abstract class.
```

Comment:
We need to mention here that `getDefault` method needs to be implemented by the provided socket factory class.

 Something like: `and provide an implementation of the static "getDefault()" method that returns an instance of the socket factory.`
The CSR needs to be updated too.

Reply:
Good point. I will amend that in both, source code and CSR.

Reply:
Done.